api/data/network-fees.js

```diff
@@ -42,7 +42,7 @@ const pollForNewFees = async () => {
       Sentry.captureException(err)
     })
   })
-  terraTaxRate = Number(Number(terraTaxRateResponse.result).toFixed(6))
```

Comment:
Oops, I assumed. But these guys are going for very accurate/long numbers

Reply:
You bet they are!

<img width="340px" src="https://user-images.githubusercontent.com/40721795/85124389-22c61980-b22a-11ea-972e-7c4324cc13d3.png">

Thank you Fabo!